README.md

```diff
@@ -119,6 +119,54 @@ Student has an attribute for the user, user has an attribute for the student, ..
 These circular references will prevent the log entry from being written.
 Including request is always safe, because the logging formatter contains dedicated code for request logging.
 
+## Javascript & CSS
+
+This project uses webpack to create javascript bundles. Custom Javascript is only added to pages when it is needed to enhance default Django functionality or to create user experience improvements.
+
+Notable examples are
+- Map Views
+- User Profile to hide unneeded blocks
+
+Javascript sources are located in frontend/src folder
+
+For local development node needs to be installed. For pure python development alternatively Docker can be used to build the javascript bundles. At the moment only production builds are supported via docker.
+
+### Build via docker
+
+Run the following commands in the main directory to setup the docker image
+```
+cd frontend
+docker build -t frontend:latest .
```

Comment:
Why don't you let `docker-compose` do the tagging? Why add another manual step at all?

Reply:
This can be improved. I took care to copy the commands I used while setting it all up in the documentation because they were helpful for me and it felt quicker this way for example when using a local runserver plus frontend building docker. 

I think we should invest quite a bit of time in documenting the whole docker setup and our design consideration. I think the current state is mergeable but not finished. But I'd like to split future improvements in smaller PRs focusing on the pain areas

Reply:
You're right I also forgot to change the README in https://github.com/match4everyone/match4everything/pull/21. I'm ok-ish with merging this but would prefer a clean version from the start